### ISSUE TITLE
SVG not showing correctly in Firefox

### ISSUE THREAD

**Issue by ricardobr001** [iconfu/svg-inject#27]
I've added the script in my project, and it's working fine in all browsers, except in Firefox, and I don't know why. My `image-clickable.svg` has clickable areas, the areas loads correctly in the correct position, but the image itself doesn't.
Nothing appear's on the console and the svg file is loading correctly. I'm using the script in an Angular app on version 6.1.4. I've tested on firefox version 64.0.2, 65.0 and Firefox Dev 66.0b3.
The problem occurs in development server, and on production.

I've added the script inside the `<head>` tag
`<script src="svg-inject.min.js"></script>`

I'm using it like the example in the README inside the `<body>`
`<img src="image-clickable.svg" onload="SVGInject(this)">`

**Comment by waruyama** [comment]
Thanks for the feedback. Is there any chance you can provide that SVG?

**Comment by ricardobr001** [comment]
The image is inside this [repo](https://gitlab.com/ricardobr001/image-svg/tree/master), name of the file is `image.svg`

**Comment by waruyama** [comment]
This seems to be a problem with ID uniquification. If you turn off ID uniquification, the SVG should display as expected. Here is wheat you need to put into the `<head>` element:

```
<head>
  <script src="svg-inject.js"></script>
  <script>
    SVGInject.setOptions({
      makeIdsUnique: false // do not make ids used within the SVG unique      
    });
  </script>
</head>
```

I do not understand why it does not work in other browsers, but in any case this appears to be a bug in the library. I will have to take a really close look at this rather complex SVG.

**Comment by waruyama** [comment]
@ricardobr001  You can take the SVG offline if it was only uploaded for this issue report.

**Comment by dasboe** [comment]
The fix is part of Version 1.2.2 which has just been released

**Comment by ricardobr001** [comment]
It's working now on version 1.2.2, Thanks for the support guys!
